lte/protos/session_manager.proto

```diff
@@ -852,6 +852,21 @@ enum SMSessionFSMState {
         RELEASED_4 = 4;
 }
 
+ // ambr_t
+ message Ambr {
+   uint64 br_ul = 1;
+   uint64 br_dl = 2;
+
+   enum BitrateUnitsAMBR {
+         BPS = 0;
+         KBPS = 1;
+         }
+   // Unit (either bps or Kbps)
+   BitrateUnitsAMBR br_unit = 3;
+
```

Comment:
spacing looks weird here.

Don't we already have an ambr struct somewhere else?